src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -38,6 +38,10 @@ import 'childnode-remove-polyfill';
 import { i18n } from '@kbn/i18n';
 import { CoreSystem } from '__kibanaCore__'
 
+// Load all Kibana styles
+const context = require.context('ui/styles', false, /[\\/\\\\](?!mixins|variables|_|\\.|bootstrap_(light|dark))[^\\/\\\\]+\\.less/);
```

Comment:
do we load only `font_awesome.less`?
probably we should add it implicitly then

Reply:
After talking to @snide, I decided to replace ui/autoload/styles with a single import to the font awesome file. This will retain the current behavior for plugins that import `ui/autoload/all` or `ui/autoload/styles` but avoid leaking this globally.